group/secp256k1/curve.go

```diff
@@ -0,0 +1,41 @@
+// Package secp256k1 is an implementation of the kyber.{Group,Point,Scalar}
+// interfaces, based on btcd/btcec and kyber/group/mod
+//
+// XXX: NOT CONSTANT TIME!
+package secp256k1
+
+import (
+	"math/big"
+
+	secp256k1BTCD "github.com/btcsuite/btcd/btcec"
```

Comment:
Why do you rename the package here? Anyway, with the inclusion of the btcec code, this will all be local and not imported.

You should include the minimum of the btcec code you can, for example not ciphering.go, etc. Probably just ScalarBaseMult and ScalarMult and their dependencies, including the pre-computed point table. Furthermore, you can leave gensecp256k1.go and genprecomps.go out, just change the comment in secp256k1.go to explain where it came from (i.e. the real btcec package).

Reply:
I just found the name btcec obscure.